FFM-7293, FFM-7296: relnotes Node 1.2.13/4 (#1089)
harness/developer-hub: release-notes/feature-flags.md

Harness Feature Flags is updated regularly in Harness SaaS. Review the notes below for details about recent changes.

:::info note
Harness deploys updates progressively to different Harness SaaS clusters. You can identify the cluster hosting your account in your Account Overview page. The features and fixes in the release notes may not be available in your cluster immediately.

Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
:::

## March 21, 2023

### What's new

This release does not include new features.

### Early access

This release does not include early access features.

### Fixed issues

#### Feature Flags SDKs

The **Node.js server SDK** has been updated to version **1.2.14** with the following updates:

> * The event source was updated to version 2.1.3. (FFM-7198)

Reviewer: ```suggestion
* The EventSource library was updated to version 2.1.3. (FFM-7198)
```

* On streaming errors, the error was not included in the `retrying` event payload. This fix adds the error to the payload. (FFM-7198)

> * Checks were added to see if errors are retryable, and if not, the SDK stops retrying. (FFM-7198)

Reviewer 2: ```suggestion
* Added checks to see if errors can be retried, and if they can't, then the SDK stops retrying. (FFM-7198)
```

I know devops involves these type of terms, like retryable, but I am not sure how well it translates for ESL readers.
